imap_processing/idex/tests/test_l1_cdfs.py

```diff
@@ -0,0 +1,91 @@
+import os
+import tempfile
+
+import numpy as np
+import pytest
+import xarray as xr
+from cdflib.xarray import cdf_to_xarray
+from cdflib.xarray.xarray_to_cdf import ISTPError
+
+from imap_processing.idex.idex_packet_parser import PacketParser
+
+
+@pytest.fixture()
+def decom_test_data():
+    return PacketParser("imap_processing/idex/tests/imap_idex_l0_20230725_v01-00.pkts")
+
+
+def test_idex_cdf_file(decom_test_data):
+    # Verify that a CDF file can be created with no errors thrown by xarray_to_cdf
+    with tempfile.TemporaryDirectory() as tmpdir:
```

Comment:
I just realized you can actually use the `tmpdir` or `tmp_path` fixture so we don't even need to do this.
https://docs.pytest.org/en/7.1.x/how-to/tmp_path.html

```suggestion
def test_idex_cdf_file(decom_test_data, tmp_path):
    # Verify that a CDF file can be created with no errors thrown by xarray_to_cdf
```